plugins/uartdmx/UartDmxThread.cpp

```diff
@@ -93,46 +105,94 @@ void *UartDmxThread::Run() {
       buffer.Set(m_buffer);
     }
 
-    if (!m_widget->SetBreak(true))
-      goto framesleep;
+    writeDmxData(buffer);
+  }
+  return NULL;
+}
 
-    if (m_granularity == GOOD)
-      usleep(m_breakt);
+/**
+ * Write the DMX data to the actual UART interface.
+ */
+void UartDmxThread::writeDmxData(const DmxBuffer &buffer) {
```

Comment:
We capitalise the first character too:
```suggestion
void UartDmxThread::WriteDmxData(const DmxBuffer &buffer) {
```

Given we've got the interface method WriteDMX perhaps we can come up with something less ambiguous here, WriteDMXToUART or something? The other equivalent classes don't seem to split it off so no real inspiration.

Reply:
I decided to stick with WriteDMXToUART